react-ticket-booking-app/src/services/concert-service.js

```diff
@@ -0,0 +1,31 @@
+import ticketBookingApiInstance from './http-interceptor'
+
+export class ConcertService {
+    ticketBookingApi = ticketBookingApiInstance;
+
+    getConcertList(concertName, concertType) {
+        var urlAdd = ''
+        if (concertName && concertName.length > 0) urlAdd += '?containsInName=' + concertName //
```

Comment:
use `searchParams`

https://stackoverflow.com/questions/486896/adding-a-parameter-to-the-url-with-javascript